tests/rbr/AdjustRegionsTest.java

```diff
@@ -0,0 +1,59 @@
+package rbr;
+
+import java.util.*;
+import org.junit.*;
+import util.*;
+
+public class AdjustRegionsTest {
+
+    private Set<Character> ip = new HashSet<>();
+    private Set<Character> op = new HashSet<>();
+    private Set<Vertex> dst = new HashSet<>();
+    private Region region;
```

Comment:
'region' is the only attribute that you use outside 'setUp()' and so should be the only attribute, don't you think so?